### DISCUSSION TITLE
path variable의 명명 케이스

### DISCUSSION THREAD

**Discussion by amaran-th** [BDD-CLUB/01-doo-re-back#54] [논의 사항]
저번에 보름님의 코드를 리뷰하다가 [patah variable의 명명 케이스를 컨벤션으로 정하면 좋을 것 같다](https://github.com/BDD-CLUB/01-doo-re-back/pull/32/files#r1464813664)는 얘기가 나왔었는데요, 함께 논의해보면 좋을 것 같습니다.
1. 카멜 케이스
2. 케밥 케이스

**Comment by shkisme** [comment]
저는 카멜 케이스로 했으면 좋겠습니다! 이유는
```java
@PutMapping("/{teamId}")
public ResponseEntity<Void> updateFeed(
        @PathVariable final Long teamId,
        @RequestBody final TeamUpdateRequest request
) {
```
이렇게 PathVariable로 값을 받아올 때 저희 컨벤션인 카멜 케이스를 따를 수 있고, 코드 작성시에도 편하다고 생각합니다. 🙂 또 JS 컨벤션도 카멜 케이스로 알고 있어서, 프론트 분들과도 명명 규칙을 맞출 수 있지 않을까 생각합니다!

**Comment by JJimini** [comment]
저도 카멜케이스가 더 편한 것 같아요! 이유는 보름님 의견과 같습니다!

**Comment by lcqff** [comment]
저도 Rest Api 디자인 가이드 찾아봤는데 일반적으로 URI는 케밥으로 하되 그 외로는 카멜 케이스를 사용하는것 같네요
개인적으로는 케밥보단 카멜이 훨씬 코드 작성할때 편해서 코드 내에서까지 케밥 케이스 사용하는건 별롭니다~!

**Comment by 02ggang9** [comment]
만장일치로 카멜로 결정되었습니다~!
